src/App.js

```diff
@@ -0,0 +1,254 @@
+  )
+}
+const Navigation = (props) => {
+  return (
+    <div className="navigation">
+      <button onClick={props.onPrev}>{'\u2B05'} Previous</button>
+      <button onClick={props.onModalClose}>Close</button>
+      <button onClick={props.onNext}>Next {'\u27A1'}</button>    
+    </div>
+  )
+}
+
+const Download = (props) => {
+  return (
+    <div className="download">
+      <button id="downloadBtn" onClick={props.onDownload} value="download">Download</button>
+    </div>
+  )
+}
+
+const Preview = (props) => {
+  return (
+    <div id="preview-all">
+      <div className="column">
+        <p>
+          <strong>PAGE {props.activeData + 1}:</strong>
+          <a href={props.oldUrl}> {props.oldUrl}</a>
+        </p>
+        <iframe title="preview-0" className="preview" src={'https://image.thum.io/get/width/1024/fullpage/allowJPG' + props.oldUrl} width="100%" height="500" />
+      </div>
+      <div className="column">
+        <p>
+          <strong>PAGE {props.activeData + 1}:</strong>
+          <a href={props.newUrl}> {props.newUrl}</a>
+        </p>
+        <iframe title="preview-1" className="preview" src={'https://image.thum.io/get/width/1024/fullpage/allowJPG' + props.newUrl} width="100%" height="500" />
+      </div>
+    </div>
+  )
+}
+
+const Modal = (props) => {
+  return (
+      <div>
+        <div id="status">
+          <span id="status-text"></span>
+          <p>
+            <strong>Status:</strong> {props.status}
+          </p>
+        </div>
+        <div className="action">
+          <button className="accept" onClick={props.onAccept}>Accept</button>
+          <button className="reject" onClick={props.onReject}>Reject</button>
+        </div>
+        <Preview {...props}></Preview>
+        <div className="skip">
+          <p>To skip to the item without status: </p>
+          <button className="skip-button" onClick={props.onSkip}>Skip</button>
+        </div>
+        <Download 
+          onDownload={props.onDownload}>
+        </Download>
+        <Navigation {...props}></Navigation>
+      </div>
+  );
+}
+
+function App() {
+  const [parsedData, setParsedData] = useState([]);
+  const [newData, setNewData] = useState([]);
+  const [activeData, setActiveData] = useState('');
+  const [selectedFile, setSelectedFile] = useState();
+	const [isFilePicked, setIsFilePicked] = useState(false);
+  const [oldUrl, setOldUrl] = useState('');
+  const [newUrl, setNewUrl] = useState('');
+  const [status, setStatus] = useState('');
+
+  const changeHandler = (event) => {
+    setSelectedFile(event.target.files[0]);
+		setIsFilePicked(true);
+    Papa.parse(event.target.files[0], {
+      header: true,
+      skipEmptyLines: true,
+      complete: function (results) {
+        setParsedData(results.data);
+        setNewData(results.data);
+      },
+    });
+
+  }
+  
+  const handleSubmission = () => {
+    setActiveData(0);
+    handleData(0);
+	}
+
+  const handleModalClose = () => {
+    setActiveData('');
```

Comment:
**Suggestion (non-blocking)**: I would avoid trying to use the mixed typing here of a number and string -- I would just make the default be `0`!

Reply:
Nice catch! Made my changes in the [Modal.js](https://github.com/Automattic/compare-snapshot-tool/pull/1/files#diff-c80a14057369079bbb58db6e6af9b9b1a67a4126ea2d5b2cea658c5cc5b80326) file. Thank you!